city_scrapers/spiders/wicks_bocc.py

```diff
@@ -0,0 +1,112 @@
+    # we scrape the embedded URL instead
+    start_urls = ["https://sedgwick.granicus.com/ViewPublisher.php?view_id=34"]
+    location = {
+        "name": "Ruffin Auditorium",
+        "address": "100 N. Broadway, Lower Level, Wichita, KS 67202",
+    }
+
+    def parse(self, response):
+        """Parse meeting items from agency website iframe."""
+
+        # get upcoming events
+        for item in response.css("table")[0].css("tbody tr"):
+            meeting = Meeting(
+                title=item.css("td")[0].css("::text").get(),
+                description="",
+                classification=COMMISSION,
+                start=self._parse_start(item),
+                end="",
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(response, item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+        # get archived events in current year. ignore all other years
+        for item in response.css(".TabbedPanelsContent")[0].css("table tbody tr"):
+            # skip if row does not have enough data cells to avoid IndexErrors
+            if len(item.css("td")) < 2:
+                continue
+
+            meeting = Meeting(
+                title=item.css("td")[0].css("::text").get(),
+                description="",
+                classification=COMMISSION,
+                start=self._parse_start(item),
+                end="",
+                all_day=False,
+                time_notes="",
+                location=self.location,
+                links=self._parse_links(response, item),
+                source=response.url,
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_start(self, item):
+        """
+        Parse start date as a naive datetime object.
+        Fetch from correct table cell and clean up.
+        """
+
+        # attempt to get date
+        try:
+            date_cell = item.css("td")[1]
+            raw_date = date_cell.css("::text").get()
+            clean_date = raw_date.strip().replace("\xa0", " ")
+            parsed_date = parse(clean_date)
+        except IndexError:
+            parsed_date = None
+
+        return parsed_date
+
+    def _parse_time_notes(self, item):
+        """
+        Parse any additional notes on the timing of the meeting.
+        If Agenda link is given, meeting start time may be found in there.
+        Eventually we could scrape the link
+        but for now we can just tell user to check link themselves.
+        """
+        agenda_url = item.css("td")[2].css("a::attr(href)").get()
+        return "Check Agenda for start time." if agenda_url else ""
```

Comment:
I think the problem is that the page has changed since the spider was originally generated. The page looks like this today, Thu, Mar 11, 2024, for me

<img width="1440" alt="Screen Shot 2024-04-11 at 1 37 58 PM" src="https://github.com/City-Bureau/city-scrapers-wichita/assets/8742021/c66586d4-0f6b-4b1c-9e32-01b55b1c4525">

There are no agenda links in the upcoming events section anymore. There used to be. They have moved from Upcoming events to Archived events. 

<img width="1440" alt="Screen Shot 2024-04-11 at 1 54 01 PM" src="https://github.com/City-Bureau/city-scrapers-wichita/assets/8742021/c2c44770-60c0-4397-80e6-68fbd7dba947">

Another problem is that currently, the loop is slightly different based on which part of the page we are iterating over. I assumed there would be no time notes for archived events (which makes sense). However, after another look, I see that some archived events still have Agenda links. Logic based on Agenda link presence doesn't make sense in this case. I will adjust the code accordingly. Maybe the strategy below works better

```
# iterate through upcoming events
#   give all of them a generic time note
#   get agenda link if any
# iterate through archived events
#   do not set time note
#   make sure to get agenda and video links
```

Reply:
Good stuff! Thanks for making those changes, @cruznunez . And yeah, I understand, re: changing site. This is one of those little gotchas with building scrapers.